src/Microsoft.SqlTools.ServiceLayer/Admin/Contracts/DatabaseInfoWrapper.cs

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.SqlTools.ServiceLayer.Admin.Contracts 
+{
+    /// <summary>
+    /// Wrapper object for the database info
+    /// </summary>
+    public class DatabaseInfoWrapper
```

Comment:
>DatabaseInfoWrapper [](start = 17, length = 19)

do we need a wrapper, or should we add properties to DatabaseInfo that read\write to the dictionary...the second approach is what we do in ConnectionDetails for example and it seems to work fine (and avoids having another class).